cocotb/regression.py

```diff
@@ -189,38 +183,45 @@ def initialise(self):
                           (valid_tests.module,
                            valid_tests.funcname))
 
+        # Process Hooks
+        ###################
+
         for module_name in self._hooks:
-            self.log.info("Loading hook from module '"+module_name+"'")
+            self.log.info("Loading hook from module '" + module_name + "'")
             module = _my_import(module_name)
 
             for thing in vars(module).values():
                 if hasattr(thing, "im_hook"):
                     try:
                         test = thing(self._dut)
                     except Exception:
-                        self.log.warning("Failed to initialize hook %s" % thing.name, exc_info=True)
+                        self.log.error("Failed to initialize hook %s" % thing.name, exc_info=True)
```

Comment:
I'd leave this as is. Hooks aren't really useful, and an `error` here would suggest propagation in a way that doesn't happen - a warning as was here before was more indicative of the fact that things continue regardless.